README.md

```diff
@@ -1,18 +1,71 @@
 # parse-hamnosys
-Sign language HamNoSys notation parsing tool.
+
+![parser schema](./imgs/schemat.JPG)
+
+The main goal of the parser is to translate a label representing a SL gloss,
+written in HamNoSys format, into a form that can be used for DL-based classification.
+We used a decision tree to decompose the notation into numerical multilabels
+for the defined classes.
+The parser analyzes a series of symbols and matches each symbol with the class
+that describes it (while assigning it the appropriate number) or removes it.
+
+## Hamburg Sign Language Notation System
+
+The Hamburg Sign Language Notation System (HamNoSys) is a _phonetic_ 
+transcription system that has been widespread for more then 20 years.
+HamNoSys does not refer to different national finger alphabets and
```

Comment:
What do you mean by „national finger alphabets”?

Reply:
specific languages, like BSL, DGS ect